module2/lessons/independent_challenge_live_review.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: IC Review
+---
+
+## Learning Goals
+
+- Interpret User Stories to write tests
+- Use TDD to implement desired functionality
+- Understand how to breakdown, approach, and complete Independent Challenges
+
+
+## The Plan
+
+Prior to this session, you will have had 2 hours to work on an Independent Challenge (IC) that is sent to you by instructors. This review will occur after the timed IC, and will cover all user stories (except for any extensions). 
+
+One instructor will demonstrate an approach to breaking down the tasks at hand, writing specs, and using those specs to guide implementation of the desired functionality.
+
+You are welcome, and encouraged, to ask questions at any point during the live review. 
```

Comment:
The IC review is going to be a video next inning followed by office hours. Could we update this sentence to indicate they can ask questions either during a live session or in the office hours after watching a video of the review?

Also, is it 2 hours or 3 hours for this first IC?